LICENSE.bin

```diff
@@ -410,6 +411,7 @@
    Mozilla Public License
 
    Javassist
+   H2 Database Engine
```

Comment:
If we only use h2 in the test code, we don't need include it in the `LICENSE.bin`.

Reply:
Done